src/search_service/config/plugins/categories/genomes.json

```diff
@@ -1,11 +0,0 @@
-{
```

Comment:
I renamed this file to old_genomes.json and the other one to refseq_genomes.json because it seemed to be confusing for the front to map those terms.